src/metabase/cmd/load_from_h2.clj

```diff
@@ -100,6 +102,8 @@
    Collection
    CollectionRevision
    DashboardFavorite
+   FieldFingerprint
+   TableFingerprint
```

Comment:
The comment below indicates that order may be important, or is it only DataMigrations that matters where in this list it's placed?